replica/replica.go

```diff
@@ -224,7 +224,11 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 	r.insertBackingFile()
 	r.ReplicaType = replicaType
 
-	PreloadLunMap(&r.volume)
+	if err := PreloadLunMap(&r.volume); err != nil {
+		logrus.Error("underlying file system does not support extent mapping")
```

Comment:
Lets also print the actual error received.

Reply:
We are returning the error to the callee, callee is printing the error : "operation not permitted". So printing the error here will pollute the logs unneccessary.

Reply:
Agreed